src/main/java/com/faforever/client/notification/TransientNotificationController.java

```diff
@@ -112,6 +113,10 @@ public void onClicked(MouseEvent event) {
     if (event.getButton().equals(MouseButton.SECONDARY)) {
       dismiss();
     } else {
+      if (!StageHolder.getStage().isFocused()) {
+        StageHolder.getStage().setIconified(false);
+        StageHolder.getStage().requestFocus();
```

Comment:
Don't call `StageHolder.getStage()` repeatedly, extract to variable (select, CTRL+ALT+V)